source/ui.js

```diff
@@ -49,6 +49,22 @@ const printCommitLog = async repoUrl => {
 	};
 };
 
+const confirmIgnoredFiles = async pkg => {
+	const ignoredFiles = await util.getNewAndUnpublishedFiles(pkg);
+	if (ignoredFiles === undefined || ignoredFiles.length === 0) {
+		return true;
+	}
+
+	const answers = await inquirer.prompt([{
+		type: 'confirm',
+		name: 'confirm',
+		message: `The following new files are not part of your published package:\n${ignoredFiles.join('\n')}\n Continue?`,
```

Comment:
Maybe style (using `chalk`) the ignored files similarly to how the commits list looks?

Reply:
Done.